Sources/TuistCore/Graph/Graph.swift

```diff
@@ -458,10 +466,11 @@ extension Graph {
     }
 
     // swiftlint:disable:next line_length
-    func frameworkUsesDynamicLinking(frameworkMetadataProvider: FrameworkMetadataProviding = FrameworkMetadataProvider()) -> (_ frameworkNode: PrecompiledNode) -> Bool {
+    func frameworkUsesDynamicLinking(
+        frameworkMetadataProvider: FrameworkMetadataProviding = FrameworkMetadataProvider()
```

Comment:
The indentation here seems odd. If you did it to make swiftlint happy I'd suggest disabling that check of this method definition.

Reply:
i think, I did it deliberately to avoid longer line but will restore it as it has the lint disabled already in place.